libipl/p10/common.C

```diff
@@ -287,3 +287,29 @@ int ipl_set_sbe_state_all_sec(enum sbe_state state)
 	}
 	return ret;
 }
+
+void ipl_plat_clock_error_handler(
+	const std::vector<std::pair<std::string, std::string>>& ffdcs_data,
+	uint8_t clk_pos)
+{
+	FFDC ffdc;
+	ffdc.ffdc_type = FFDC_TYPE_HWP;
```

Comment:
This is not the HWP/HWP plat error right? I think it is used before calling the HWP right? 

Do we need to add a new type or no needs?

Reply:
i think we need new type.
Are you ok for FFDC_TYPE_IPL?

Reply:
What about FFDC_TYPE_PLAT?

Reply:
Ok

Reply:
libipl is a plat layer, right? (To have BMC specific/platform-specific logic in the libipl)? 
I think you are referring PLAT from ekb infra.

are you going to handle any special logic in the application based on the FFDC_Type? If not, you can use the same and we can change it later if needed.

Reply:
done